_data/people.yaml

```diff
@@ -350,13 +347,19 @@ alexwlchan:
   pronouns: they/she
   twitter: alexwlchan
   website: https://alexwlchan.net
-ali-humayun:
+alihumayun:
   bio: I am 23 years old and currently working at a sports media company as an editor.
-    I graduated in October 2020 from UCL in MSc Digital Humanities and have a previous
-    background in history. I am extremely interested in the intersection between technology,
-    the law and society. Also, I'm very keen to help address educational inequity
-    as well as learning more about community building.
+    I graduated in October 2020 from UCL in MSc Digital Humanities, with a year of experience
+    in journalism. I also have a previous background in history, and I am about to embark on a
+    postgraduate diploma in law, with hopes of pursuing a career at the intersection of law and
+    technology. Also, I'm very keen to help address educational inequity as well as learning more
+    about community building and the open science community!
   city: London
+  country: United Kingdom
+  expertise:
+  - Editing
+  - Legal
+  - Diversity issues
   first-name: Ali
   github: false
```

Comment:
Can you remove this one there?

Reply:
Yes, I can.

Two things I don't understand in life:
1. How I missed that line.
2. Why Github doesn't notify me anymore.